src/python/doc/rips_complex_tflow_itf_ref.rst

```diff
@@ -0,0 +1,39 @@
+:orphan:
+
+.. To get rid of WARNING: document isn't included in any toctree
+
+TensorFlow layer for Vietoris-Rips persistence
+##############################################
+
+.. include:: differentiation_sum.inc
+
+Example of gradient computed from Vietoris-Rips persistence
+-----------------------------------------------------------
+
+.. testcode::
+
+    from gudhi.tensorflow import RipsLayer
+    import tensorflow as tf
+
+    X = tf.Variable([[1.,1.],[2.,2.]], dtype=tf.float32, trainable=True)
+    rl = RipsLayer(maximum_edge_length=2., dimensions=[0])
+
+    with tf.GradientTape() as tape:
+        dgm = rl.call(X)
```

Comment:
Thanks for catching this. It is actually `rl.call(X)[0][0]` since the output is a list of list (first list is dimension, second list is [finite, essential]). I've fixed it.